spec/mailers/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+RSpec.describe UserMailer, type: :mailer do
```

Comment:
Please add proper test for this...
Try to test the subject, to, from and body of the email sent.